dist/partials/query.editor.html

```diff
@@ -27,8 +27,8 @@
 
     <div class="gf-form" ng-hide="!ctrl.target.shouldOverrideGranularity">
       <div class="gf-form-select-wrapper max-width-15">
```

Comment:
@mhshimul if you can remove the select wrapper, it will remove the dropdown icon. I've rested the changes, they are good to go otherwise.

Reply:
select wrapper removed.